### ISSUE TITLE
0.4.0 not compatible with ruby 1.9.3

### ISSUE THREAD

**Issue by aerostitch** [chetan/simplecov-console#7]
Hi,

FYI the version 0.4.0 (since merge of PR #6) of the gem is not compatible with ruby 1.9.3. I get:
```
`require': /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/simplecov-console-0.4.0/lib/simplecov-console.rb:8: unknown type of %string (SyntaxError)
  ATTRIBUTES = %i(table_options)
                  ^
```
Please update the gemfile accordingly.

Thanks for your help! :)
Joseph

**Comment by chetan** [comment]
Fixed by #8

**Comment by chetan** [comment]
Pushed v0.4.1 with the fix
